broker/readme.md

```diff
@@ -0,0 +1,4 @@
+```
```

Comment:
The readme is still odd, see https://github.com/neondatabase/neon/pull/2733#discussion_r1011764691 
I think, current commit message is better than this readme 🙂 

> Compilation now requires protoc to be installed

This belongs to the main readme, where we list all things needed for compilation: https://github.com/neondatabase/neon/blame/c4b417ecdbc030f245673362dc23fbf4f279180e/README.md#L37

I think we should change all installation instructions, for Mac, the package is named `protobuf-c`